.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,20 @@
+name: 📚 Documentation
+description: Report an issue related to https://pytorch.org/docs/stable/index.html
+
+body:
+- type: textarea
+  attributes:
+    label: 📚 The doc issue
+    description: >
+      A clear and concise description of what content in https://pytorch.org/docs/stable/index.html is an issue. If this has to do with the general https://pytorch.org website, please file an issue at https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this has to do with https://pytorch.org/tutorials, please file an issue at https://github.com/pytorch/tutorials/issues/new.
```

Comment:
Emmm. The description here is incorrect for TorchData. And, we only have inline doc for now.

Reply:
I will update it and let users know that we only have inline docs for now and the web version is coming soon.